packages/core/bootstrap/src/lib/external-adapter/logger.ts

```diff
@@ -1,23 +1,65 @@
-import { v4 as uuidv4 } from 'uuid'
-import { createLogger, format, transports } from 'winston'
-const { combine, timestamp, json, prettyPrint } = format
+import { uuid } from '../util'
+import pino from 'pino'
 
-// We generate an UUID per instance and add it to the logs
-const uuid = () => {
-  if (!process.env.UUID) process.env.UUID = uuidv4()
-  return process.env.UUID
-}
+export const paths = [
```

Comment:
This should be exposed by the WS module, not here.

Reply:
Moved the paths to `ws/config` and reimported them here.
Future redaction paths can have the specific paths written in the module, and imported to the logger module

Reply:
Awesome, thx!